backend/src/main/java/springweb/backend/service/ItemService.java

```
@@ -41,4 +42,11 @@ public List<Item> getAllItems() {
     public Item getItemById(String id) {
         return itemRepository.findById(id).orElse(null);
```

May be you should throw exception if item id not found similar to your exception in delete function.